roles/network_scripts/templates/ifcfg.j2

```diff
@@ -0,0 +1,81 @@
+{% if 'block' in item %}
+{{ item['block']|string }}
+{% endif %}
+{% if 'BONDING_OPTS' in item %}
```

Comment:
Would a loop be handy here, with all the templated VAR_NAMES in some vars array (`ifcfg_vars` used in this suggestion)?

```ini
{% for VAR in ifcfg_vars %}
{% if VAR in item %}{{ VAR }}="{{ item[VAR] | string }}"{% endif %}
{% endfor %}
```

From a maintenance standpoint, though, I'm not sure if I prefer the loop approach or just writing it all out, since it's *way* simpler, if slightly tedious, to come in here and just fix what needs fixing in the unlikely event that new and interesting things find their way into ifcfg scripts. I'm indifferent. :)

Reply:
I'm wishy-washy on the difference there. I like explicit. Because it's an easy place to come reference and because it avoids someone going:
```
networks:
  DERPDERP_MC_DERP: 123
```

And then going, New Issue -> "It's broke"

But I also like generic because it allows a user to go, "I know more than Greg does" and do the things they want.